ftd/src/executor/element.rs

```diff
@@ -83,6 +111,18 @@ pub struct Column {
     pub common: Common,
 }
 
+#[derive(serde::Deserialize, Debug, Default, PartialEq, Clone, serde::Serialize)]
+pub struct HTMLData {
+    pub title: ftd::executor::Value<Option<String>>,
+}
```

Comment:
@Arpita-Jaiswal: Do we need this type? I think we should merge it with `Document`. I do not want to mention `HTML` in our core type hierarchy as we are supposed to agnostic of rendering layer.